### DISCUSSION TITLE
Sel set

### DISCUSSION THREAD

**Discussion by foreverfreedown** [deepmodeling/deepmd-kit#466] [Q&A]
I have trained the Si potenial with the train data supplied by Deepmd-kit website,and I use this potenial to calculate the single vacancy formation energy of Si.But my results are not match with DFT results.Then,I change the sel set and I found the results change a lot.So I want to know how to set the sel and whether the data provided is enough.I hope other can solve my questions.Thanks a lot!

**Comment by njzjz** [comment]
See #108
